code/src/components/Error.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import ButtonBack from './ButtonBack';
+
+const Error = () => {
+  return (
+    <>
```

Comment:
Do you really need to use this <> -tag? You have a section as parent element

Reply:
Exactly, no need for that, should be removed